src/mailstoreobserver.cpp

```diff
@@ -321,7 +323,11 @@ void MailStoreObserver::updateNotifications()
         initNotification(notification);
         notification->setAppName(properties.first);
         notification->setAppIcon(properties.second);
-        notification->setHintValue("x-nemo-feedback", "email_exists");
+        if (!feedbackSet) {
+            // just set this once to ensure we don't play multiple tones etc
+            notification->setHintValue("x-nemo-feedback", "email_exists");
+            feedbackSet = true;
```

Comment:
nit: since the flag is always set and not otherwise used, the change might as well be done at start of the block

Reply:
Sure.